grisera_api/graph_api_service.py

```diff
@@ -55,7 +55,7 @@ def create_properties(self, node_id: int, node_model: BaseModel):
         request_body = [{"key": key, "value": value} for key, value in node_dict.items()
                         if value is not None and key != 'additional_properties']
 
-        if node_dict['additional_properties'] is not None:
+        if 'additional_properties' in node_dict and node_dict['additional_properties'] is not None:
```

Comment:
Is that needed? I'm not sure about it since 'additional_properties' default is None

Reply:
Yes, you are right. It was a temporary solution for experiment_model without additional_properties.